ui/pages/confirm-send-token/confirm-send-token.container.js

```diff
@@ -37,21 +41,23 @@ const mapDispatchToProps = (dispatch) => {
       });
 
       dispatch(
-        updateSend({
-          from,
+        editTransaction({
+          id: id?.toString(),
           gasLimit,
           gasPrice,
-          gasTotal: null,
-          to,
+          from,
           amount: tokenAmountInHex,
-          errors: { to: null, amount: null },
-          editingTransactionId: id?.toString(),
-          token: {
-            ...tokenProps,
-            address: tokenAddress,
-          },
+          address,
+        }),
+      );
+
+      dispatch(
+        updateSendAsset({
```

Comment:
This is async but isn't being `await`-ed. Not sure if this was intentional or not, or whether this could lead to issues :thinking: At the very least it's worth a comment if it is intentional though.

Reply:
the edit functionality on send-token isn't currently "wired up" but between @adonesky1 and your feedback, I did make some changes that I'll be merging in soon-ish which moves to have this be awaited (but it is moved into the send slice).